database/database.js

```diff
@@ -0,0 +1,9 @@
+const Pool = require('pg').Pool
```

Comment:
This is used for the credentials, for setting up the server locally. Nice configuration, we might want to remove these next time when committing.